uber/config.py

```diff
@@ -215,10 +248,10 @@ def SUPPORTER_COUNT(self):
         with sa.Session() as session:
             attendees = session.query(sa.Attendee)
             individual_supporters = attendees.filter(sa.Attendee.paid.in_([self.HAS_PAID, self.REFUNDED]),
-                                                     sa.Attendee.amount_extra >= self.SUPPORTER_LEVEL).count()
```

Comment:
We may want to convert this back, depending on what `SUPPORTER_COUNT` is actually used for.